src/main/java/org/folio/rest/impl/HoldingsStorageAPI.java

```diff
@@ -408,4 +431,60 @@ private boolean isUUID(String id) {
       return false;
     }
   }
+
+  private CompletableFuture<Void> updateItemEffectiveCallNumbersByHoldings(AsyncResult<SQLConnection> connection, PostgresClient postgresClient, HoldingsRecord holdingsRecord) {
+    CompletableFuture<Void> future = new CompletableFuture<>();
+    Criterion criterion = new Criterion(
+      new Criteria().addField("holdingsRecordId")
+        .setJSONB(false).setOperation("=").setVal(holdingsRecord.getId()));
+    postgresClient.get(ITEM_TABLE, Item.class, criterion, false, false, response -> {
```

Comment:
I think this get should use the transaction connection. If it fails for some reason we need to roll back the transaction.

Reply:
I originally was using the connection, but the results weren't be supplied until it completed and it was the cause of a lot of timeouts. Since it's just a get it doesn't make any changes to roll back in the case of a failure so it should be safe.